stories/core/Table.stories.tsx

```diff
@@ -314,14 +314,17 @@ Sortable.args = {
 
 export const Filters: Story<TableProps> = (args) => {
   const { columns, data, ...rest } = args;
-  const onClickHandler = (
-    props: CellProps<{
-      name: string;
-      description: string;
-      ids: number[];
-      date: Date;
-    }>,
-  ) => action(props.row.original.name)();
+  type TableStoryDataType = {
+    index: number;
+    name: string;
+    description: string;
+    ids: number[];
+    date: Date;
+  };
```

Comment:
This story code is getting too complicated. Can't we infer this type using `typeof something`?

Reply:
I can't use `typeof` because data is array, maybe I could move it outside.

Reply:
leave it I guess. Moving outside would not make sense as every story differs